zerocopy-derive/tests/ui-nightly/enum.rs

```diff
@@ -72,9 +72,17 @@ enum TryFromBytes1 {
 }
 
 #[derive(TryFromBytes)]
-#[repr(u8)]
 enum TryFromBytes2 {
-    A(u8),
+    A,
+    B(u8),
+}
+
+struct NotTryFromBytes;
+
+#[derive(TryFromBytes)]
+#[repr(u8)]
+enum TryFromBytes3 {
+    A(NotTryFromBytes),
 }
```

Comment:
It looks like this isn't generating any error, presumably because the entire compilation is failing during an earlier compiler pass. You will likely need to put this in a separate file.